CHANGES.md

```diff
@@ -1,5 +1,17 @@
 ## Changelog
 
+### HEAD
+ * Compatibility: allow pandas version newer than 1.0, less than 2.0
```

Comment:
why "newer than 1.0"?  How is that constraint made?  
if it is made, then do we still need to support pandas 0.22 for the old medusa/pegasus environments?

Reply:
I have adjusted the message: update pinning from pandas<1.1, to pandas<2

we're still testing on python 3.6 with pandas 0.x